client/grpc-web-client/src/client.ts

```diff
@@ -261,7 +262,13 @@ class GrpcClient<TRequest extends ProtobufMessage, TResponse extends ProtobufMes
     this.started = true;
 
     const requestHeaders = new Metadata(metadata ? metadata : {});
-    requestHeaders.set("content-type", "application/grpc-web+proto");
+
+    if (this.props.format === "text") {
+      requestHeaders.set("content-type", "application/grpc-web-text");
+      requestHeaders.set("accept", "application/grpc-web-text");
+    } else {
+        requestHeaders.set("content-type", "application/grpc-web+proto");
+    }
     requestHeaders.set("x-grpc-web", "1"); // Required for CORS handling
 
     this.transport.start(requestHeaders);
```

Comment:
I assume the client work here is not completed? I don't see any base64 encoding/decoding of payloads.

Reply:
No this is my WIP, tbh it's going a slightly bigger refactor than I first hoped as the transports need a fair bit of work - @johanbrandhorst shall we break that out into a separate PR and just have this one be to introduce the server-side changes?

Reply:
Well, it would be cool to get this in sooner rather than later so @evanj can get on with the work, if we're happy to wait with the automated tests until the frontend is done, I don't think that's a bad idea!

Reply:
@evanj lets remove this from this PR to avoid blocking while @jonnyreeves looks at what would need to be done. I imagine we can ship this to master and when we have the functionality in the frontend we can release a minor version bump.

Reply:
Removed!